testutil/mockbot.go

```diff
@@ -156,6 +154,10 @@ func (md *MockDaemon) worker(n int) {
 	}
 }
 
+func mustString(s string, _ error) string {
+	return s
```

Comment:
panic if err!=nil? otherwise we could be hiding pretty nasty errors.

or, if you want to ignore the error, call this something else like `tryString`.